docs/usage_tips/interface_mappings.md

```
@@ -25,13 +25,13 @@ Example:
 ![Interface Mappings Page](../img/interface_mappings/interfacemappings_menu.png){ width="250" }
 
 * From the main menu, navigate to the Plugins section
-* Under Netbox Librenms Plugin, Select "Interface Mappings"
+* Under Netbox Librenms Plugin, select "Mappings" — Interface Mappings is the first tab
 
 #### Creating a New Mapping:
 
 ![](../img/interface_mappings/addmapping.png){ width="50" }
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)